### ISSUE TITLE
 BNR.getRates is not a function

### ISSUE THREAD

**Issue by SharpBCD** [JorgenVatle/bnr-exchange-history#5]
Me again. :-)

Running 4.1.1. on the NODE 20. Tried with the 3.00 as well but same result. The code is fine, was running for more than 1 year daily without problems. 
Now I get this error and not sure how to fix it. Any tips on your side? I gotta solve this quick and get to eFactura where the real pain is. :-)
```
    curs = await BNR.getRates({
      date,
      invoice: false
    })
```
Error:  TypeError: BNR.getRates is not a function

**Comment by JorgenVatle** [comment]
Hey there. Sorry this got buried in my GitHub notifications. I will have a look later in the evening and get back to you with more details.

I suspect it didn't handle the transition into 2024 well 😅

**Comment by JorgenVatle** [comment]
Looks like we were missing a default export for the package entrypoint. This should now be fixed with [`bnr-exchange-history@4.1.6`](https://www.npmjs.com/package/bnr-exchange-history)

If you run into any other issues, do let me know. ✌️
